### ISSUE TITLE
Request for version that dosen't contain "Font size change feature"

### ISSUE THREAD

**Issue by wiaalwial** [TekkenOverlay/TekkenOverlay#169]
I'm having the "Fatal Error" problem. Some says that removing all the other overlay programs will solve it, or updating VGA driver will, but non of them worked for me sadly.

The only solution that had worked for me was the minor patch version KulaGGin uploaded on this thread ▼
https://github.com/TekkenOverlay/TekkenOverlay/issues/140
(of course only when the game version was 1.0 too)

I am very grateful for your hard work and do not want to burden you further, but this is the only way for me. I am wondering if you can upload this minor patch again (and if you don't mind, continuously) corresponding to the new version of overlay and game.

English is not my first language. There may be awkward expressions. Please understand.

**Comment by KulaGGin** [comment]
See if this version works for you:
[TekkenOverlay-7.5.11-8.1.02.01-1-SNAPSHOP-0-[2024-03-03].zip](https://github.com/TekkenOverlay/TekkenOverlay/files/14473716/TekkenOverlay-7.5.11-8.1.02.01-1-SNAPSHOP-0-.2024-03-03.zip)

This is a version with the feature disabled by default.

**Comment by wiaalwial** [comment]
> See if this version works for you: [TekkenOverlay-7.5.11-8.1.02.01-1-SNAPSHOP-0-[2024-03-03].zip](https://github.com/TekkenOverlay/TekkenOverlay/files/14473716/TekkenOverlay-7.5.11-8.1.02.01-1-SNAPSHOP-0-.2024-03-03.zip)
> 
> This is a version with the feature disabled by default.

This perfectly works for me! Fianlly I can use this amazing mod agian.... I really appreciate it.

It seems that just one simpe change of setting could solve my problem. I also checked that if I copy and paste the file "Tekken8OverlaySetting.yaml" to the folder of original version of 1.02.01, then no crash occurs too. 

Will this setting file compatible with future version of overlay? If so, I wouldn't need to bother you with another request for another patch I think.

**Comment by KulaGGin** [comment]
> Will this setting file compatible with future version of overlay? If so, I wouldn't need to bother you with another request for another patch I think.

Yes, it's the way I'll release the Overlay from now on, so hopefully the crashes won't happen for people.

**Comment by wiaalwial** [comment]
> Yes, it's the way I'll release the Overlay from now on, so hopefully the crashes won't happen for people.

I got it. Thank you :)
